client/typescript/fine-uploader.d.ts

```diff
@@ -2691,6 +2693,14 @@ declare module "fine-uploader" {
 
 }
 
+declare module "fine-uploader/lib/all" {
+    import { azure } from 'fine-uploader/lib/azure';
+    export * from 'fine-uploader/lib/core';
```

Comment:
@bradleyayers I think this should be `export * from 'fine-uploader'` instead of core

Reply:
Ah interesting. So do you think `azure` and `s3` should be exported from `fine-uploader`?

Reply:
`azure` and `s3` is fine here, It's just that we are exporting the core `fine-uploader` from here, it should be the UI module which is `fine-uploader` and not `fine-uploader/lib/core`

Reply:
Makes sense, and do you think `azure` and `s3` should *also* be exported from `fine-uploader`? I thought they were accessible via `qq.s3.*` etc

Reply:
`azure` and `s3` should be exported from the `fine-uploader/lib/all` module but not from the `fine-uploader` module.
And `FineUploaderBasic` isn't exported from `fine-uploader`.
That must be imported from `fine-uploader/lib/core`

Reply:
I think since we are not exporting `FineUploaderBasic` from `fine-uploader` module, we shouldn't export the same from `fine-uploader/lib/azure` and `fine-uploader/lib/s3` as well.
The core modules should only be exported from the `fine-uploader/lib/core`, `fine-uploader/lib/core/azure` and `fine-uploader/lib/core/s3`

That's what makes more sense to me.

Reply:
Okay, I've removed the FineUploaderBasic export from the non-core lib modules, and switched to `export *` as suggested.

Reply:
Great, I think we're good to go